src/drivers/gps/gps.cpp

```diff
@@ -511,6 +567,73 @@ int GPS::setBaudrate(unsigned baud)
 	return 0;
 }
 
+bool GPS::fileExist(const char *filename)
+{
+	struct stat buffer;
+	return stat(filename, &buffer) == 0;
```

Comment:
It runs on Snappy if I comment out that line.

Is there an alternative to `stat()` that might work on QURT?